ref/supported-video-codecs.md

```diff
@@ -5,11 +5,9 @@ Not all codecs are supported by one or more of the supported GPU models, neither
 Currently, the following video codecs are supported:
 
  * H.264 - The use of H.264 requires a license from the [MPEG LA](https://www.mpegla.com/). Ensure you have the rights to stream H.264 encoded video content to your users.
+ * AV1 - Depending on the GPU model used in the deployment, AV1 hardware encoding is the default preference over H.264 on supported GPUs, such as NVIDIA Ada Lovelace Architecture-based GPUs like L4. Otherwise, it will fallback to AV1 software encoding.
  * VP8
 
-In the future we plan to add support for:
-
- * VP9
- * AV1
-
+In the future we plan to add support for VP9 as well.
```

Comment:
Lets drop this one. We currently do not plan anymore to eventually support VP9. With AV1 being supported there should be no need.